src/Ladybug_SunPath.py

```diff
@@ -509,8 +509,9 @@ def main(latitude, longitude, timeZone, elevation, north, hour, day, month, time
                     listInfo[i][5] = (1,1,1)
                     listInfo[i][6] = (12,31,24)
                     #find the numbers
-                    for h, hr in enumerate(sunUpHours):
-                        value = selList[int(math.floor(hr))] + (selList[int(math.ceil(hr))] - selList[int(math.floor(hr))])* (hr - math.floor(hr))
+                    for h, hr in enumerate(sunUpHours):                     
+#                        value = selList[int(math.floor(hr))] + (selList[int(math.ceil(hr))] - selList[int(math.floor(hr))])* (hr - math.floor(hr))
+                        value = selList[int(hr-1)]

```

Comment:
@chriswmackey,
I checked for the city of Barrow (Alaska, USA) in the arctic circle. Please see the following image.
![barrow_21june_midnight](https://cloud.githubusercontent.com/assets/20086924/25174462/1146440a-2515-11e7-907a-0d4deb3e4910.JPG)
For the antarctic circle, we only have the weather data for the gateway cities. I am not sure if it would be right to use the weather data from those cities to check the midnight sun phenomenon.